### DISCUSSION TITLE
Getting search box to include other items on menu

### DISCUSSION THREAD

**Discussion by jennyberry** [thuliteio/doks#864] [Q&A] [answered]
I did a research and follow these steps in this https://github.com/h-enk/doks/discussions/395?sort=new#discussioncomment-1168082
But it seems doesnt work on my project.
This is my link to my repo: https://github.com/jennyberry/doks.git

Thanks a lot!

**Comment by h-enk** [comment, marked as answer]
Update `searchSectionsShow` + `searchSectionsIndex ` — see also: https://github.com/jennyberry/c3eth-new/blob/master/config/_default/params.toml#L103-L104

Documentation is underway — for now, see https://github.com/h-enk/doks/pull/799
